java/opendcs/src/main/java/opendcs/dai/LocationLevelDAI.java

```diff
@@ -0,0 +1,89 @@
+package opendcs.dai;
+
+import decodes.tsdb.DbIoException;
+import opendcs.dao.LocationLevelDAO.LocationLevelValue;
```

Comment:
You've now created a circular dependency of an interface on it's implementation, this is always a terrible idea.

Move the LocationLevelValue and Spec to a separate package space dedicated to just being data.